src/Searchable/Aggregator.php

```diff
@@ -187,7 +187,11 @@ public static function makeAllSearchable()
             $softDeletes =
                 in_array(SoftDeletes::class, class_uses_recursive($model)) && config('scout.soft_delete', false);
 
-            $instance->newQuery()->when($softDeletes, function ($query) {
+            $instance->newQuery()
+            ->when(true, function ($query) use ($instance) {
+                $instance->makeAllSearchableUsing($query);
```

Comment:
This currently fails, since `makeAllSearchableUsing` is not available to call outside of the class: it's defined as a `protected` method in Laravel Scout. As long as it's protected, we can't use this method here.

Reply:
Done!

https://github.com/laravel/scout/pull/577

Thank you!